client/src/common/components/Comments/Comment.jsx

```diff
@@ -1,42 +1,24 @@
 import React, { PureComponent } from 'react';
 import PropTypes from 'prop-types';
 import Linkify from 'react-linkify';
-import classNames from 'classnames';
 
-import { UserIcon } from 'assets/images/icons';
 import { dateFromString } from 'common/utils/helpers';
-import UserIconPlaceholder from 'assets/images/user.svg';
+import Avatar from 'common/components/Avatar';
 
 class Comment extends PureComponent {
-  state = {
-    isAvatarError: false
-  };
-
-  handleError = () => this.setState({ isAvatarError: true });
-
   render() {
     const { comment } = this.props;
     const { authorName, authorAvatarUrl, createdAt, text } = comment;
     const date = dateFromString(createdAt);
-    const { isAvatarError } = this.state;
 
     return (
       <div className="comment">
-        <div
-          className={classNames('comment__avatar', {
-            'comment__avatar--error': isAvatarError
-          })}
-        >
-          {authorAvatarUrl ? (
-            <img
-              alt={`${authorName} avatar`}
-              className="comment__image"
-              onError={this.handleError}
-              src={isAvatarError ? UserIconPlaceholder : authorAvatarUrl}
-            />
-          ) : (
-            <UserIcon />
-          )}
+        <div className="comment__avatar">
+          <Avatar
+            avatarUrl={authorAvatarUrl}
+            className="activity__image"
```

Comment:
wrong class name.
According to BEM children can be nested only within the parent.
Where is `.activity` parent?

This should be raather independent Block element, so `.activity-avatar` or something